### PR TITLE
4.7.4 update

### PR DESCRIPTION
**Pull request by yinlinchen**

### CODE REVIEW

File: 4.7.4/config/fedora_camel_toolbox.script [hunk 1]
```diff
@@ -0,0 +1,9 @@
+feature:repo-add mvn:org.fcrepo.camel/toolbox-features/4.6.1/xml/features
```
Comment (awoods) [inline 4.7.4/config/fedora_camel_toolbox.script]: This should probably be using 4.7.2.
https://github.com/fcrepo4-exts/fcrepo4-vagrant/blob/master/install_scripts/fedora_camel_toolbox.script#L3

File: 4.7.4/config/test.ttl [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@prefix :      <http://base/#> .
```
Comment (awoods) [inline 4.7.4/config/test.ttl]: For configuring inferencing, see:
https://github.com/fcrepo4-exts/fcrepo4-vagrant-base-box/blob/master/config/test.ttl

**Review by awoods — CHANGES_REQUESTED** [review]
Pending response to review comments.

File: 4.7.4/config/fedora_camel_toolbox.script (hunk 1 above)
Comment (yinlinchen) [reply 4.7.4/config/fedora_camel_toolbox.script]: @awoods  I usually sync the software stack versions with https://github.com/fcrepo4-exts/fcrepo4-vagrant as per Fedora 4 release. The released 4.7.4 version using 4.6.1, see https://github.com/fcrepo4-exts/fcrepo4-vagrant/blob/fcrepo4-vagrant-4.7.4/install_scripts/fedora_camel_toolbox.script. I will update the 4.7.2 along with Fedora 4.7.5 release.

**Review by awoods — APPROVED** [review]

**Pull request merged**